dolphin/api/v1/storages.py

```diff
@@ -171,9 +173,22 @@ def _is_registered(self, context, access_info):
         access_info_dict.pop('model', None)
 
         # Check if storage is registered
-        if db.access_info_get_all(context,
-                                  filters=access_info_dict):
-            return True
+        access_info_list = db.access_info_get_all(context,
+                                                  filters=access_info_dict)
+        if access_info_list:
+            access_info = access_info_list.pop()
```

Comment:
We expect  to get the storage_id only  from the last element from the list?

Reply:
only one element shoul be in the list according to the filters. But one more scenario we need concern: if user use different extra_attributes in the requrest body when registering same storage.

I will update this pr to solve this situation.

Reply:
done